pylib/cqlshlib/cql3handling.py

```diff
@@ -1501,6 +1505,32 @@ def alter_type_field_completer(ctxt, cass):
                        ;
 '''
 
+syntax_rules += r'''
+<slaName> ::= <identifier>
+             | <quotedName>
+             | <unreservedKeyword>
+             ;
+
+<createSlaStatement> ::= "CREATE" "SERVICE_LEVEL" <slaName>
+                              ( "WITH" <slaProperty> ("AND" <slaProperty>)*)?
+                        ;
+
+<alterSlaStatement> ::= "ALTER" "SERVICE_LEVEL" ("IF" "EXISTS")? <slaName>
+                              ( "WITH" <slaProperty> ("AND" <slaProperty>)*)?
+                       ;
+
+<slaProperty> ::= "WORKLOAD_TYPE" "=" <stringLiteral>
+                 | "TIMEOUT" "=" <mapLiteral>
+                 | "SHARES" "=" <boolean>
```

Comment:
SHARES is `integer`

Reply:
fix to a whole number, i.e. share can't be negative, right ?

Reply:
From documnetation:
```
shares_number - The number of shares of the resources you’re granting to the service level name. You can use any number within the range from 1 to 1000
```